.github/workflows/locale_update_all.yaml

```diff
@@ -26,16 +26,23 @@ jobs:
       RAILS_ENV: i18n
       SKIP_TEST_RESET: true
       SKIP_DATABASE_SETUP: true
+      BUNDLER_GROUPS: "manageiq_default,ui_dependencies,qpid_proton"
     steps:
     - uses: actions/checkout@v4
     - name: Set up system
       run: bin/before_install
+    - name: Install qpid dependency - libcurl4-openssl-dev
+      run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev
+    - name: Install qpid dependency - libqpid-proton11-dev
+      run: sudo apt-add-repository --yes 'deb http://us.archive.ubuntu.com/ubuntu lunar universe' && sudo apt-get install -t lunar -y libqpid-proton11-dev
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: "3.0"
         bundler-cache: true
       timeout-minutes: 30
+    - name: Set bundle config with qpid_proton
+      run: bundle config with qpid_proton
```

Comment:
I was able to fix the locale_update_all workflow by installing qpid_proton, adding it to the BUNDLER_GROUPS so it's auto-required, and setting the `bundle `--with qpid_proton`.

https://github.com/jrafanie/manageiq/actions/runs/6659788595/job/18099615071

Reply:
setup-ruby installs all the gems, so I think this is too late.  I think BUNDLE_WITH env var can be set